### ISSUE TITLE
Allow arranging icons by extension

### ISSUE THREAD

**Issue by countingpine** [mate-desktop/caja#151]
Arranging icons by MIME type is surely useful in many cases, but sometimes it's useful to sort by the extension itself.

Not only does it sort the files in an (arguably) more predictable way, but would be particularly useful in cases such as browsing a folder of source code, where you can have files with many different extensions (.c, .h, .txt etc), but they all have the same detected "type" (plain text document), and cannot be separated out from each other.

Long story short: sometimes the extension better identifies the kind of file than the MIME type.  It would be nice to have this as a sorting option.
##

**Comment by muesli4** [comment]
This should be closed as it has been implemented with: https://github.com/mate-desktop/caja/commit/0585258dc117b50cf8e8f6b5361fd815cc292cca
